scripts/settings.js

```diff
@@ -88,6 +91,10 @@ export class Settings {
      * @type {number} The decimals to display for the wallet balance
      */
     displayDecimals;
+    /**
+     * @type {Boolean} Whether Advanced Mode is enabled or disabled
```

Comment:
```suggestion
     * @type {boolean} Whether Advanced Mode is enabled or disabled
```
Really minor nit: Sometimes using class typings instead of primitives types (i.e. Boolean instead of boolean) causes problems with the linter